contracts/token/ERC20.cairo

```diff
@@ -218,18 +218,24 @@ func transfer{
         syscall_ptr : felt*, 
         pedersen_ptr : HashBuiltin*,
         range_check_ptr
-    }(recipient: felt, amount: Uint256):
+    }(recipient: felt, amount: Uint256) -> (is_success: felt):
     let (sender) = get_caller_address()
     _transfer(sender, recipient, amount)
-    return ()
+
+    tempvar is_success = 1
+    return (is_success)
```

Comment:
```suggestion
    return (1)
```

why not this?

Reply:
On it! I was just thinking of making it explicit for readability

Reply:
I see. Maybe a comment stating that `1` means `true` in Cairo